src/goto-programs/goto_convert.cpp

```diff
@@ -1258,6 +1258,7 @@ void goto_convertt::convert_switch(
 
   goto_programt tmp_cases;
 
+  size_t case_number=1;
```

Comment:
Can we get the actual case numbers, e.g. switch(...) { case 2: ... case 3: ... } will number 2 -> 1, 3 -> 2 at the moment